client/src/services/auth/authService.js

```diff
@@ -0,0 +1,42 @@
+import requestService from '@/services/requestService';
+
+const authService = {
+    async signIn(userLoginData) {
+        try {
+            const response = await requestService.post(
+                '/auth/login',
+                userLoginData
+            );
+            this.saveToken(response.access_token);
+            return response;
+        } catch (error) {
```

Comment:
this try catch and neighbor one are useless, the logic will be the same without them